cql3/statements/batch_statement.cc

```diff
@@ -215,10 +215,10 @@ void batch_statement::verify_batch_size(query_processor& qp, const std::vector<m
                     mutations.size(), fmt::join(ks_cf_pairs, ", "), size, type, threshold, size - threshold);
         };
         if (size > fail_threshold) {
-            _logger.error(error("FAIL", fail_threshold).c_str());
+            _logger.error("{}", error("FAIL", fail_threshold).c_str());
```

Comment:
Please push separately, I like submodule updates to be in the top-level.

Reply:
sure. will extract the treewide one out.

Reply:
extracted into #16612. i will drop it from this PR once the build passes.